overview/content-types/shell-command.nix

```diff
@@ -0,0 +1,49 @@
+{
+  name,
+  config,
+  lib,
+  ...
+}:
+let
+  inherit (lib) mkOption types;
+in
+{
+  options = {
+    prompt = mkOption {
+      type = types.str;
+    };
+    input = mkOption {
+      type = types.lines;
+    };
+    stdout = mkOption {
+      type = with types; nullOr lines;
+      default = null;
+    };
+    stderr = mkOption {
+      type = with types; nullOr lines;
+      default = null;
+    };
+    __toString = mkOption {
+      type = with types; functionTo str;
+      readOnly = true;
+      default = self: ''
+        <div class="code-block">
```

Comment:
Likely no need to wrap it in a div, because whatever style is applied to `div.code-block` we can just apply to `pre.shell`.